modules/sdk-unified-wallet/src/types.ts

```diff
@@ -21,3 +21,11 @@ export type CurveType = 'Ecdsa' | 'Eddsa';
 export interface GenerateUnifiedWalletOptions extends GenerateWalletOptions {
   curve: CurveType;
 }
+
+export type PaginationOrder = 'ASC' | 'DESC';
+
+export interface PaginationOptions {
+  prevPage?: number;
+  limit?: number;
+  order: PaginationOrder;
```

Comment:
what is this order by? wallet balance?

Reply:
Order by is the type of ordering (Ascending vs Descending). This is one of the parameters the `WP` [API requires](https://github.com/BitGo/bitgo-microservices/blob/904ca39df91bbe2f8048dc9a8b39380734f40c6d/packages/wallet-platform/app/controllers/api/v2/coins/features/EvmWallet.ts#L122)

Reply:
Ascending vs descending of what? it seems like wallet id?

Reply:
According to @karankurbur , by default `mongo` returns by order of creation. So it seems like the ordering is based on creation order.